module/callback_handlers.py

```diff
@@ -9,11 +9,12 @@
 # Needed to correctly run functions using globals()
 from module.utils.multi_lang_utils import get_on_demand_text
 
-from module.commands.esami import *  # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.lezioni import *  # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.help import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.drive_contribute import * # pylint: disable=wildcard-import,unused-wildcard-import
-from module.commands.aulario import * # pylint: disable=wildcard-import,unused-wildcard-import
+import logging  # pylint: disable=wildcard-import,unused-wildcard-import
```

Comment:
remove `# pylint: disable=wildcard-import,unused-wildcard-import` where you can

Reply:
If I remove those, pylint won't give a score of 10 and the deploy will fail

are you suggesting to replace this solution with a smoother one, or to straight up remove these imports?